src/NuGet.Core/NuGet.Commands/RestoreCommand/Utility/NoOpRestoreUtilities.cs

```diff
@@ -44,6 +44,12 @@ private static string GetBuildIntegratedProjectCacheFilePath(RestoreRequest requ
             return null;
         }
 
+        /// <summary>
+        /// cacheRoot + "project.nuget.cache"
+        /// </summary>
+        /// <param name="cacheRoot">Path to cache root</param>
+        /// <param name="projectPath">Not used</param>
+        /// <returns></returns>
```

Comment:
missing description.

Reply:
When it's obvious, like "Get file path", you can just assume it's the file path in the string. I'm fine with leaving those blank.
If anything, explain what `null` means or `empty` means, etc.

Reply:
Updated documentaiton